### DISCUSSION TITLE
No boards found in project

### DISCUSSION THREAD

**Discussion by SBS-rclifford** [ankitpokhrel/jira-cli#345] [Q&A] [answered]
Hi,
I am trying to list the boards in my current project using the `jira board` command. I know for sure that boards exist, but the respone is always "No boards found". I am not sure what I could be doing wrong. Is there a log file or verbose mode that I can enable to help debug my issue?

In case it makes any difference, I have an on-premise Jira installation.

```
c:\utils\jira_0.3.0_windows_amd64\bin>jira board
⠧ Fetching boards in project COMP...
✗ No boards found in project "COMP"
```

Thanks.

**Comment by ankitpokhrel** [comment]
Hi @SBS-rclifford, that could be some permission issue. Could you please check if you get boards when you query the Jira API directly?

```sh
curl --request GET \
  --url 'https://<host>/rest/agile/1.0/board?projectKeyOrId=<project_key>' \
  --user "<username>:${JIRA_API_TOKEN}"
```

**Comment by SBS-rclifford** [comment]
Hi @ankitpokhrel,
I get a sensible response back. All 3 boards in the project are recognized.
I am the project admin and also a system admin, just in case that helps to narrow down the possible issue.

```
$ curl --request GET \
--url 'https://<host>/rest/agile/1.0/board?projectKeyOrId=COMP'
--user "<username>:${JIRA_API_TOKEN}"
{"maxResults":50,"startAt":0,"isLast":true,"values":[{"id":392,"self":"https://<host>/rest/agile/1.0/board/392","name":"Kanban1","type":"kanban"},{"id":393,"self":"<host>/rest/agile/1.0/board/393","name":"Kanban2","type":"kanban"},{"id":394,"self":"https://<host>/rest/agile/1.0/board/394","name":"Kanban3","type":"kanban"}]}
```

**Comment by ankitpokhrel** [comment]
@SBS-rclifford I see that the key `total` is missing in your response. Could you please provide details on what version of Jira you are using?

**Reply by SBS-rclifford** [reply]
Yes, this is Jira version 8.2.2.

**Reply by ankitpokhrel** [reply]
This seems like an old version. I tested with v8.17+ and it is working fine. Is this the only issue you are facing in your installation?

**Reply by SBS-rclifford** [reply]
From what I can tell, yes.

**Reply by ankitpokhrel** [reply]
@SBS-rclifford could you please check if #360 fixes the issue for you? You can install it with the following command (requires go1.16+)

```sh
go install github.com/ankitpokhrel/jira-cli/cmd/jira@issue-345
```

**Reply by SBS-rclifford** [reply]
@ankitpokhrel that looks to have fixed it. I can now see all 3 boards in my default project. Thank you for looking into this issue!

**Comment by ankitpokhrel** [comment, marked as answer]
This change is available in the new release [v1.0.0-beta](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.0.0-beta)
